src/Magnum/GL/Buffer.cpp

```diff
@@ -28,6 +28,7 @@
 
 #include <tuple>
```

Comment:
This include could get wrapped in an `#ifdef MAGNUM_BUILD_DEPRECATED`.

Reply:
Done